lib/providers/portion/PortionProvider.ts

```diff
@@ -39,6 +38,33 @@ export class PortionProvider implements IPortionProvider {
     return tokenOutCurrencyAmount.multiply(new Fraction(portion.bips, 10000));
   }
 
+  getPortionQuoteAmount(
+    portionAmountToken: CurrencyAmount<Currency>,
+    quoteCurrencyAmount: CurrencyAmount<Currency>,
```

Comment:
Feel like this naming schema is kinda confusing. Why is one `XCurrencyAmount` and the other `XAmountToken`? Would `portionAmount` and `quoteAmount` make more sense?